addons/project/models/project.py

```diff
@@ -1305,6 +1305,7 @@ def message_new(self, msg, custom_values=None):
         customer_ids = [p.id for p in self.env['mail.thread']._mail_find_partner_from_emails(tools.email_split(defaults['email_from']), records=task) if p]
         partner_ids += customer_ids
         task.message_subscribe(partner_ids)
+        task._compute_allowed_user_ids()
```

Comment:
I think that the explanation is right [here](https://github.com/odoo/odoo/blob/6b084c7ace17260eb3e41f2840190a36e75eb358/addons/project/models/project.py#L883-L884)
By calling the compute here, which solve the issue, you actually run the code highlighted in the link.
I think it would be best to take the logic and apply it here directly to the partner_ids prior to call `task.message_subscribe(partner_ids)` instead of calling the compute.
What do you think?
If I remember correctly the right system has been changed somewhere between 14.0 and 15.0. Can you check? If it is the case, can you check that the issue is still happening in 15.0? If not, let me know so that we prevent the forward bot to run.

Reply:
In V15, the field visible is not there anymore. The way it works is adding the customer/portal to the followers. So, yes we should prevent the forward port.

Reply:
```I think it would be best to take the logic and apply it here directly to the partner_ids prior to call task.message_subscribe(partner_ids) instead of calling the compute.```
I'll try this approach